framework/console/controllers/AssetController.php

```diff
@@ -569,7 +569,9 @@ public function combineJsFiles($inputFiles, $outputFile)
         $content = '';
         foreach ($inputFiles as $file) {
             $content .= "/*** BEGIN FILE: $file ***/\n"
+                . "(function(){\n"
                 . file_get_contents($file)
```

Comment:
It's better to check if trimmed contents end with `;` and add it if it's not there.